arangod/RocksDBEngine/RocksDBCollection.cpp

```diff
@@ -1224,6 +1224,15 @@ Result RocksDBCollection::performUpdateOrReplace(
     }
   }
 
+  if (newSlice.length() <= 1 && isUpdate) {  // TODO move above ?!
```

Comment:
The TODO is obsolete here.
Please also leave a comment why we are exiting early here, and why we are comparing the length of `newSlice` to 1 and not 0. That makes it easier in the future to understand this code part.
```suggestion
  if (newSlice.length() <= 1 && isUpdate) {
```